### ISSUE TITLE
Custom Archive Display Conditions for CPT

### ISSUE THREAD

**Issue by WeDoStories** [elementor/elementor#9257]
## Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
When creating custom post types with CPT UI to separate different types of posts, i.e blog posts, job adverts, company news etc. I can't assign different archive designs to each CPT.  

**Describe the solution you'd like**
Add CPT option to display condition in the same way as you have tags and categories. 

**Describe alternatives you've considered**
Tags and categories, but this leaves the backend messy and unorganised.

**Comment by pingram3541** [comment]
We can add our own custom post types and enable the ability to edit them individually via Elementor Settings (post type check boxes) but we can't use the Theme Builder Display Conditions to wrap ALL posts of said time in a Theme Builder Template.

**Comment by manoj-identixweb** [comment]
Hi Everyone,
Hope you are doing great.
I am wondering if Elementor allows us to create a separate archive template using theme builder instead of only WordPress post for each post type created using CPT.

**Comment by pingram3541** [comment]
@manoj-identixweb Yes, the theme builder allows you to also build custom archive templates and assign them to specific post types, so for example you can have many different CPT defined and create different archive templates using the Theme Builder.
